### PR TITLE
General Cleanup 4

### PR DESCRIPTION
**Pull request by hensldm**

### CODE REVIEW

**Review by jpburnett — APPROVED** [review]
Late spring cleaning is nice

**Review by tom-overton — APPROVED** [review]

**Review by EllipticEllipsis — COMMENTED** [review]
This actually addresses a few things I was thinking about, although there's a few more I had some thoughts about making more consistent.

File: src/overlays/actors/ovl_En_GirlA/z_en_girla.c [hunk 1]
```diff
@@ -502,7 +502,7 @@ void EnGirlA_BuyShieldMirror(PlayState* play, EnGirlA* this) {
     func_801159EC(-play->msgCtx.unk1206C);
 }
 
-// Fanfare is handled by ovl_en_ossan
+// Fanfare is handled by the shopkeeper
```
[the remark this answers is not in the document]
Comment (hensldm) [reply src/overlays/actors/ovl_En_GirlA/z_en_girla.c:505]: Do you want me to include all four shopkeeper actors?

File: src/overlays/actors/ovl_En_Osn/z_en_osn.c [hunk 2]
```diff
@@ -29,19 +29,31 @@ const ActorInit En_Osn_InitVars = {
 };
 
 static AnimationInfo sAnimations[] = {
```
[the remark this answers is not in the document]
Comment (hensldm) [reply src/overlays/actors/ovl_En_Osn/z_en_osn.c:31]: Will deal with this in #872

File: src/overlays/actors/ovl_En_Osn/z_en_osn.h [hunk 3]
```diff
@@ -10,7 +10,7 @@ typedef void (*EnOsnActionFunc)(struct EnOsn*, PlayState*);
 typedef struct EnOsn {
     /* 0x0000 */ Actor actor;
     /* 0x0144 */ ColliderCylinder collider;
-    /* 0X0190 */ SkelAnime skelAnime;
+    /* 0x0190 */ SkelAnime skelAnime;
```
[the remark this answers is not in the document]
Comment (hensldm) [reply src/overlays/actors/ovl_En_Osn/z_en_osn.h:13]: There are a lot of these. Will probably make this part of cleanup 6 instead.

File: src/overlays/actors/ovl_En_GirlA/z_en_girla.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (EllipticEllipsis) [reply src/overlays/actors/ovl_En_GirlA/z_en_girla.c:505]: Hmm, this is starting to sound more like a file-wide note at this point.

[the remark this answers is not in the document]
Comment (hensldm) [reply src/overlays/actors/ovl_En_GirlA/z_en_girla.c:505]: The thing is it only applies to this function.

[the remark this answers is not in the document]
Comment (EllipticEllipsis) [reply src/overlays/actors/ovl_En_GirlA/z_en_girla.c:505]: Well, in that this is meant to be used with shopkeeper actors, doesn't work by itself.

[the remark this answers is not in the document]
Comment (hensldm) [reply src/overlays/actors/ovl_En_GirlA/z_en_girla.c:505]: Ah, yeah that is what `EnGirla` is meant for. I don't think they can do much by themselves.

[the remark this answers is not in the document]
Comment (hensldm) [reply src/overlays/actors/ovl_En_GirlA/z_en_girla.c:505]: I do have plans to do `OBJECT_TRT` with some cleanup of that actor, plus hopefully any last little bit of cleanup for all the shopkeepers/enGirla. I can add a note than, though I might end up changing the function name. Need to think on it.

File: src/code/z_eff_footmark.c [hunk 4]
```diff
@@ -77,7 +77,7 @@ void EffFootmark_Update(PlayState* play) {
     for (footmark = play->footprintInfo, i = 0; i < 100; i++, footmark++) {
         if (footmark->actor != NULL) {
             if ((footmark->flags & 1) == 1) {
-                if (footmark->age < 0xFFFFu) { // TODO replace with MAX_U16 or something
+                if ((u32)footmark->age < 0xFFFF) { // TODO replace with MAX_U16 or something
```
Comment (EllipticEllipsis) [inline src/code/z_eff_footmark.c:80]: We have `UINT16_MAX` now, so could actually do this now.

**Review by EllipticEllipsis — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**